typeql/language/update.feature

```diff
@@ -67,7 +67,7 @@ Feature: TypeQL Update Query
       delete $x has name "Alex";
       insert $x has name "Bob";
       """
-    Then for each session, transaction closes
+    Then session transaction closes
```

Comment:
ok last question - why is this the only non-reasoning one that is changed?

Reply:
Ah I removed this for consistency, I believe it's the only place we use `for each session` outside the transaction features.